### ISSUE TITLE
would be a way to setup sesTemplatesConfigFile in the command line?

### ISSUE THREAD

**Issue by ericmaicon** [haftahave/serverless-ses-template#13]
Hey there, 
I have 2 AWS accounts which each has a different folder of templates. 

To do so, I would like to change sesTemplatesConfigFile when I run `sls ses-template deploy`. Is it possible?

**Comment by Enase** [comment]
Unfortunately deploy command doesn't support `--config` option that allows to pass config file location. But you have two options:

1. Configure destination folder with serverless parameters:
```
custom:
    accountId: !Ref AWS::AccountId
    [yourAccountId1]:
        configPath: 'path/1'
    [yourAccountId2]:
        configPath: 'path/2'
    sesTemplatesConfigFile: ${self:custom.${self:custom.accountId}.configPath}
```
OR
```
custom:
    accountId: !Ref AWS::AccountId
    sesTemplatesConfigFile: './folderName/${self:custom.accountId}/file.js'
```
2. Use separate serverless configuration files:
see https://serverless.com/framework/docs/providers/aws/cli-reference/deploy/
`--config or -c Name of your configuration file, if other than serverless.yml|.yaml|.js|.json.`

3. Or you can wait for next release, I can add this feature.

**Comment by Enase** [comment]
Actually I don't see any reason to implement it, there are tons of workarounds. If you don't want to commit your account id, you can use your own option for that, like this:

serverless.yml example:
```
custom:
    configId1:
        configPath: 'path/1'
    configId2:
        configPath: 'path/2'
    sesTemplatesConfigFile: ${self:custom.${opt:option_name_you_prefer}.configPath}
```
command example:
`sls ses-template deploy --option_name_you_prefer configId1`

**Comment by Enase** [comment]
I will close the issue tomorrow, if you don't have any questions.

**Comment by ericmaicon** [comment]
Hey @Enase , sorry about the delay. I had hard times and I had to postpone it.

I am not sure if I am doing something wrong, but using vars doesn't work for me. Seems the yml didn't parse it:
```
custom:
  test:
      configPath: './template/test/index.js'
  sesTemplatesConfigFile: ${self:custom.test.configPath}
  sesTemplatesRegion: "eu-west-1"
```

`sls ses-template deploy`

![image](https://user-images.githubusercontent.com/1010637/70517391-7267b200-1b30-11ea-9255-50f86c86641b.png)

I did a workaround by using my CI to add the line into the serverless.yml 

Thanks

**Comment by Enase** [comment]
Unfortunately I cannot reproduce. The next part of the code works perfect for me:
```
custom:
  test:
    configPath: './template/test/index.js'
  sesTemplatesConfigFile: ${self:custom.test.configPath}
```
Could you provide exact copy of your serverless.yml or any other example that I can use to reproduce the issue?

**Comment by Enase** [comment]
BTW: make sure u use the last version. it was the bug with props in previous version: https://github.com/haftahave/serverless-ses-template/issues/12

**Comment by Enase** [comment]
Ability to pass template configuration file path as CLI argument named `sesTemplateConfig` [added](https://github.com/haftahave/serverless-ses-template/commit/f86e9e50c04f07371b4505ba085d6d8b3888dec3)
release coming soon

**Comment by Enase** [comment]
Feel free to use new published version [1.2.6](https://github.com/haftahave/serverless-ses-template/releases/tag/1.2.6)
